local-test-configuration/monitoring/index.html

```diff
@@ -741,52 +797,46 @@ <h2>Active data feeds</h2>
       const mid = Math.floor(arr.length / 2);
 
       const nums = [...arr].sort((a, b) => {
-        if (a.lt(b)) return -1;
-        else if (a.gt(b)) return 1;
+        if (a < b) return -1;
+        else if (a > b) return 1;
         else return 0;
       });
 
-      return arr.length % 2 === 0 ? nums[mid - 1].add(nums[mid]).div(2) : nums[mid];
+      return arr.length % 2 === 0 ? (nums[mid - 1] + nums[mid]) / 2n : nums[mid];
     };
 
     const decodeBeaconValue = (encodedBeaconValue) => {
       // Solidity type(int224).min
-      const INT224_MIN = ethers.BigNumber.from(2).pow(ethers.BigNumber.from(223)).mul(ethers.BigNumber.from(-1));
+      const INT224_MIN = 2n ** 223n * -1n;
       // Solidity type(int224).max
-      const INT224_MAX = ethers.BigNumber.from(2).pow(ethers.BigNumber.from(223)).sub(ethers.BigNumber.from(1));
+      const INT224_MAX = 2n ** 223n - 1n;
```

Comment:
Very cool, I didn't realise there was a native BigInt or that it could be specified like this.

Reply:
Yeah, in v6 of ethers they switched the BigNumber library to native BigInt so we have the advantage of some of the built in operators.

Reply:
I suspect it will be much more efficient for processing too.